numpy/random/mtrand/mtrand.pyx

```diff
@@ -4533,6 +4547,179 @@ cdef class RandomState:
         x.shape = tuple(final_shape)
         return x
 
+    def complex_normal(self, loc=0.0, gamma=1.0, relation=0.0, size=None):
+        """
+        complex_normal(loc=0.0, gamma=1.0, relation=0.0, size=None)
+
+        Draw random samples from a complex normal (Gaussian) distribution.
+
+        Parameters
+        ----------
+        loc : complex or array_like of complex
+            Mean of the distribution.
+        gamma : float, complex or array_like of float or complex
```

Comment:
`gamma` is Hermitian, hence real in the 1d case. It is also required to be >=0 as it is >= `|relation|`.